runtime-decompiler/src/main/java/org/jrd/frontend/MainFrame/FiletoClassValidator.java

```diff
@@ -30,20 +30,20 @@ public FiletoClassValidator(JLabel validation, JTextField filePath, JTextField c
 
     @Override
     public void insertUpdate(DocumentEvent e) {
-        work(e);
+        work();
```

Comment:
Does all those still override? I have my doubts....

Reply:
@judovana only the insertUpdate(), removeUpdate() & changedUpdate() methods are overridden, and thus their signature don't change. work() is our private method, which did not use the DocumentEvent parameter at all, so that could be removed.